bin/bb/cmd/migrate_test.go

```diff
@@ -0,0 +1,110 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/bytebase/bytebase/resources/mysql"
+
+	// embed expected output
+	_ "embed"
+)
+
+var (
+	//go:embed testdata/expected/migrate_test_TestMigrate_01
+	_TestMigrate01 string
+	//go:embed testdata/expected/migrate_test_TestMigrate_02
+	_TestMigrate02 string
+	//go:embed testdata/expected/migrate_test_TestMigrate_03
+	_TestMigrate03 string
+)
+
+func TestMigrate(t *testing.T) {
+	mysql, stop := mysql.SetupTestInstance(t, 23306)
```

Comment:
organize the ports in a central place?

https://github.com/bytebase/bytebase/blob/main/tests/tests.go#L108

Reply:
done